gke-whereami/README.md

```diff
@@ -0,0 +1,85 @@
+# gke-whereami
+
+A simple Kubernetes-oriented app for describing the location of the pod serving a request via its attributes (cluster name, cluster region, pod name, namespace, service account, etc). The response payload includes an emoji that is hashed from the pod name, which makes it a little easier for a human to visually identify the pod you're dealing with.
+
+This was originally written for testing & debugging multi-cluster ingress use cases on GKE (now Ingress for Anthos).
+
+### Setup
+
+Create a GKE cluster 
+
+First define your environment variables (substituting where #needed#):
+
+```
+export PROJECT_ID=#YOUR_PROJECT_ID#
+
+export COMPUTE_REGION=#YOUR_COMPUTE_REGION#
+
+export CLUSTER_NAME=whereami
+```
+
+Now create your resources:
+
+```
+gcloud beta container clusters create $CLUSTER_NAME \
+  --enable-ip-alias \
+  --enable-stackdriver-kubernetes \
+  --region=$COMPUTE_REGION \
+  --num-nodes=1 \
```

Comment:
re: backend overlay example, I recommend the cluster has at least 2 nodes

Reply:
it's provisioning a regional cluster, so it should result in a node count of 3.

Reply:
via `--region=$COMPUTE_REGION`

Reply:
discussed with @askmeegs and it's a validation issue with `gcloud`